src/Components/Main/Pages/UsersPage/UsersPageContainer.tsx

```diff
@@ -14,27 +14,29 @@ import {
   followErrorsSelector,
   unfollowErrorsSelector,
 } from '../../../../redux/selectors/users';
+import { useTypedSelector } from '../../../../hooks/useTypedSelector';
+import { useTypedDispatch } from '../../../../hooks/useTypedDispatch';
 
-const UsersPageContainer = () => {
-  const users = useSelector(usersSelector);
-  const page = useSelector(pageSelector);
-  const pageSize = useSelector(pageSizeSelector);
-  const totalCount = useSelector(totalCountSelector);
-  const subscribersId = useSelector(subscribersIdSelector);
-  const isFetchingUsers = useSelector(isFetchingUsersSelector);
-  const usersError = useSelector(usersErrorSelector);
-  const followErrors = useSelector(followErrorsSelector);
-  const unfollowErrors = useSelector(unfollowErrorsSelector);
+const UsersPageContainer: FC = (): ReactElement | boolean => {
```

Comment:
![image](https://github.com/MaksimkaSota/social-network/assets/18385315/383af22e-253b-49f5-8362-6c1dfcc9562f)

Reply:
I do not have this problem...  

![image](https://github.com/MaksimkaSota/social-network/assets/94923150/dd657cfb-e8e6-49c8-a4f8-c1bc105fe3a3)